code/modules/jobs/job_types/map_changes/map_changes.dm

```diff
@@ -1,4 +1,6 @@
 //this needs to come after the job_types subfolder to keep the correct ordering
 
 #include "..\..\..\..\_maps\map_files\Pahrump-AB\job_changes.dm"
+#include "..\..\..\..\_maps\map_files\Baltimore\job_changes.dm"
 #undef JOB_MODIFICATION_MAP_NAME
+ 
```

Comment:
linters say this is missing a trailing newline, but i see it there, try adding an empty line with nothing in there at the end